src/main/java/org/openhab/binding/zigbee/converter/ZigBeeConverterColorColor.java

```diff
@@ -38,8 +39,23 @@
     private HSBType currentHSB;
     private ZclColorControlCluster clusterColorControl;
     private ZclLevelControlCluster clusterLevelControl;
+    private ZclOnOffCluster clusterOnOff;
 
     private boolean initialised = false;
+    private PercentType lastBrightness = PercentType.HUNDRED;
+    private Object colorUpdateSync = new Object();
+
+    private boolean supportsHue = false;
```

Comment:
I added it like this because at some moment it maight be worth adding CT to this channel, instead of keeping it in a different one (in the end everything is in the color control cluster), but I will change it to its original implementation (I only had put supportsHue at the beginning).

Reply:
Ok, if CT gets added here then we can update, but at the moment CT is still a separate channel (I think) in other bindings, and currently in the ZigBee binding a converter can't handle more than 1 channel so it can't handle CT and Color in one converter.